src/assets/css_properties

```diff
@@ -552,4 +653,7 @@ wrap-flow
 wrap-inside
 wrap-through
 writing-mode
+x
+y
 z-index
+zoom
```

Comment:
Missing newline.

Reply:
Is it going to break?

Reply:
I guess not, but it's a good practice to have files with trailing newline.

Reply:
I don't think so, it's just code style: I have it setup that it always adds a new line on save if there is not one.

Reply:
@gdotdesign It's more than that, see https://thoughtbot.com/blog/no-newline-at-end-of-file for more in-depth explanation.

Reply:
Dam it Unix.